doc/nrf/config_and_build/configuring_app/zephyr_samples_sysbuild.rst

```diff
@@ -6,42 +6,46 @@ Using Zephyr samples with sysbuild
+|               ``SB_CONFIG_NETCORE_RPC_HOST``            | :ref:`ble_rpc_host`                       | ✓         | ✕        | Requires that application be setup for this mode                                                                      |
++---------------------------------------------------------+-------------------------------------------+-----------+----------+-----------------------------------------------------------------------------------------------------------------------+
+|               ``SB_CONFIG_NETCORE_802154_RPMSG``        | :ref:`zephyr:nrf-ieee802154-rpmsg-sample` | ✕         | ✓        |                                                                                                                       |
++---------------------------------------------------------+-------------------------------------------+-----------+----------+-----------------------------------------------------------------------------------------------------------------------+
+|               ``SB_CONFIG_NETCORE_MULTIPROTOCOL_RPMSG`` | :ref:`multiprotocol-rpmsg-sample`         | ✓         | ✓        |                                                                                                                       |
++---------------------------------------------------------+-------------------------------------------+-----------+----------+-----------------------------------------------------------------------------------------------------------------------+
+|               ``SB_CONFIG_NETCORE_IPC_RADIO``           | :ref:`ipc_radio`                          | ✓         | ✓        | Requires additional configuration. The following Kconfigs provide predefined configurations:                          |
+|                                                         |                                           |           |          | ``SB_CONFIG_NETCORE_IPC_RADIO_BT_HCI_IPC`` enables HCI serialization                                                  |
+|                                                         |                                           |           |          | for Bluetooth, ``SB_CONFIG_NETCORE_IPC_RADIO_BT_RPC`` enables nRF RPC serialization for Bluetooth, or                 |
+|                                                         |                                           |           |          | ``SB_CONFIG_NETCORE_IPC_RADIO_IEEE802154`` enables spinel serialization for IEEE 802.15.4.                            |
++---------------------------------------------------------+-------------------------------------------+-----------+----------+-----------------------------------------------------------------------------------------------------------------------+
+|               ``SB_CONFIG_NETCORE_NONE``                | No image                                  | -         | -        |                                                                                                                       |
```

Comment:
```suggestion
|               ``SB_CONFIG_NETCORE_NONE``                | No image                                  | n/a         | n/a        |                                                                                                                       |
```
(`-` render as bullet points in live docs)

Reply:
Could use --- as an option as well. That will render as a single dash in the live doc.